bin/install_pywps.sh

```diff
@@ -0,0 +1,196 @@
+electronicMailAddress=Email Address
+providerSite=http://pywps.org
+phoneVoice=+xx-xxx-xxx-xxxx
+phoneFacsimile=+xx-xxx-xxx-xxxx
+administrativeArea=Administrative Area
+
+[server]
+maxoperations=50
+maxinputparamlength=1024
+maxfilesize=3mb
+tempPath=/tmp
+debug=true
+EOF
+
+echo 'creating WSGI wrapper'
+
+cat << EOF > "$PYWPS_WSGI"
+import os
+import pywps
+from pywps.Exceptions import NoApplicableCode, WPSException
+
+
+def application(environ, start_response):
+
+    os.environ['PYWPS_CFG'] = environ['PYWPS_CFG']
+    os.environ['PYWPS_PROCESSES'] = environ['PYWPS_PROCESSES']
+
+    status = '200 OK'
+    response_headers = [('Content-type', 'text/xml')]
+    start_response(status, response_headers)
+
+    inputQuery = None
+    if "REQUEST_METHOD" in environ and environ["REQUEST_METHOD"] == "GET":
+        inputQuery = environ["QUERY_STRING"]
+    elif "wsgi.input" in environ:
+        inputQuery = environ['wsgi.input']
+
+    if not inputQuery:
+        err = NoApplicableCode("No query string found.")
+        return [err.getResponse()]
+
+    # create the WPS object
+    try:
+        wps = pywps.Pywps(environ["REQUEST_METHOD"])
+        if wps.parseRequest(inputQuery):
+            pywps.debug(wps.inputs)
+            wps.performRequest()
+            return wps.response
+    except WPSException as e:
+        return [e]
+    except Exception as e:
+        return [e]
+EOF
+
+echo 'creating PyWPS processes'
+
+cat << EOF > "$PYWPS_PROCESSES/__init__.py"
+__all__ = ['hello_world']
+EOF
+
+cat << EOF > "$PYWPS_PROCESSES/hello_world.py"
+from pywps.Process import WPSProcess
+class HelloWorldProcess(WPSProcess):
+    def __init__(self):
+        WPSProcess.__init__(
+            self,
+            version='0.1.0',
+            identifier='hello-world',
+            title='Hello World',
+            abstract='Sample process',
+            storeSupported=False,
+            statusSupported=False)
+
+        self.data = self.addComplexInput(identifier='name',
+                                         title='Name')
+
+        self.out = self.addComplexOutput(identifier='output',
+                                         title='Output')
+
+    def execute(self):
+        value = self.data.getValue()
+        self.out.setValue('Hello World from %s' % value)
+        return
+EOF
+
+echo 'creating Apache configuration'
+
+cat << EOF > "$PYWPS_APACHE_CONF"
+WSGIScriptAlias /pywps/wps.py $PYWPS_WSGI
+<Location /pywps/wps.py>
+  SetEnv PYWPS_CFG $PYWPS_CFG
+  SetEnv PYWPS_PROCESSES $PYWPS_PROCESSES
+</Location>
+EOF
+
+echo 'creating desktop launcher'
+
+cat << EOF > "$PYWPS_DESKTOP"
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=PyWPS
+Comment=PyWPS
+Categories=Application;Education;Geography;WPS
+Exec=firefox $PYWPS_URL?service=WPS&version=1.0.0&request=GetCapabilities
+Icon=/usr/local/share/icons/$ICON_NAME
+Terminal=false
+StartupNotify=false
+Categories=Education;Geography;
+EOF
+
+cp $PYWPS_DESKTOP "$USER_HOME/Desktop/Web Services"
```

Comment:
The Desktop folder "Web Services" does not exist at this point in the build.
The desktop launcher should just land in "$USER_HOME/Desktop/"